src/EVM.hs

```diff
@@ -1643,9 +1641,8 @@ cheatActions =
                 _ -> vmError (BadCheatCode sig)
               _ -> vmError (BadCheatCode sig)
           else
-            let msg = encodeUtf8 "ffi disabled: run again with --ffi if you want to allow tests to call external scripts"
-            in vmError . Revert . ConcreteBuf $
-              abiMethod "Error(string)" (AbiTuple . V.fromList $ [AbiString msg]),
+            let msg = "ffi disabled: run again with --ffi if you want to allow tests to call external scripts"
+            in partial $ UnexpectedSymbolicArg vm.state.pc msg [],
```

Comment:
This was not showing up for failing `prove` tests since we stopped treating `revert` as a failure there, switching it to `partial` surfaces the error again.